dlt/_workspace/cli/_runtime_command.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+import time
+from typing import Optional
+
+from dlt._workspace._workspace_context import active
+from dlt._workspace.auth import AuthService
+from dlt._workspace.exceptions import (
+    RuntimeNotAuthenticated,
+    WorkspaceIdMismatch,
+    LocalWorkspaceIdNotSet,
+)
+from dlt._workspace.runtime_clients import AUTH_BASE_URL
```

Comment:
I think these URLs should be part of the runtime config sections in config.toml. The associated class is `RuntimeConfiguration`. This way you can point the runtime commands to various endpoints for development or staging environments. The defaults can stay the localhost for now.